.yarnrc.yml

```
@@ -0,0 +1 @@
+yarnPath: .yarn/releases/yarn-1.22.0.js
```

Should we bump this version since this PR has been outstanding for a little while and yarn has had a few patch releases since then?

```suggestion
yarnPath: .yarn/releases/yarn-1.22.5.js
```